scripts/print_track_docstring.py

```diff
@@ -46,9 +46,9 @@ def get_attributes_and_properties(class_instance):
         else:
             raise ValueError("Unknown type {}".format(attr))
 
-    non_attributes = list(itertools.chain.from_iterable(
-        [properties, cached_properties, functions]
-    ))
+    non_attributes = list(
+        itertools.chain.from_iterable([properties, cached_properties, functions])
```

Comment:
maybe it's a good idea to remove from PR files which are not related to the dataset, even if it's just formatting. it makes it more cumbersome to check for errors on large files.